### ISSUE TITLE
PHP Warning when trying to sync user group as contacts

### ISSUE THREAD

**Issue by mathieuser** [putyourlightson/craft-campaign#431]
Hi Ben and co!

I'm getting a `[yii\base\ErrorException](https://www.yiiframework.com/doc-2.0/yii-base-errorexception.html)` when i'm trying to use a user group as contacts.

Stack Trace:

```
yii\base\ErrorException: foreach() argument must be of type array|object, null given in /var/www/html/vendor/putyourlightson/craft-campaign/src/controllers/ImportsController.php:312
Stack trace:
#0 /var/www/html/vendor/craftcms/cms/src/web/ErrorHandler.php(79): yii\base\ErrorHandler->handleError(2, 'foreach() argum...', '/var/www/html/v...', 312)
#1 /var/www/html/vendor/putyourlightson/craft-campaign/src/controllers/ImportsController.php(312): craft\web\ErrorHandler->handleError(2, 'foreach() argum...', '/var/www/html/v...', 312)
#2 /var/www/html/vendor/putyourlightson/craft-campaign/src/controllers/ImportsController.php(190): putyourlightson\campaign\controllers\ImportsController->_getImportModelFromParams('field_')
#3 [internal function]: putyourlightson\campaign\controllers\ImportsController->actionImportUserGroup()
#4 /var/www/html/vendor/yiisoft/yii2/base/InlineAction.php(57): call_user_func_array(Array, Array)
#5 /var/www/html/vendor/yiisoft/yii2/base/Controller.php(178): yii\base\InlineAction->runWithParams(Array)
#6 /var/www/html/vendor/yiisoft/yii2/base/Module.php(552): yii\base\Controller->runAction('import-user-gro...', Array)
#7 /var/www/html/vendor/craftcms/cms/src/web/Application.php(305): yii\base\Module->runAction('campaign/import...', Array)
#8 /var/www/html/vendor/craftcms/cms/src/web/Application.php(608): craft\web\Application->runAction('campaign/import...', Array)
#9 /var/www/html/vendor/craftcms/cms/src/web/Application.php(284): craft\web\Application->_processActionRequest(Object(craft\web\Request))
#10 /var/www/html/vendor/yiisoft/yii2/base/Application.php(384): craft\web\Application->handleRequest(Object(craft\web\Request))
#11 /var/www/html/www/index.php(12): yii\base\Application->run()
#12 {main}
```

The error seems to origin here:

```
{
        $import = new ImportModel();
 
        $import->unsubscribe = (bool)$this->request->getBodyParam('unsubscribe');
        $import->forceSubscribe = (bool)$this->request->getBodyParam('forceSubscribe');
        $import->emailFieldIndex = $this->request->getBodyParam('emailFieldIndex');
        $import->fieldIndexes = $this->request->getBodyParam('fieldIndexes');
 
        if ($fieldIndexPrefix) {
            foreach ($import->fieldIndexes as $key => $fieldIndex) {
                if ($fieldIndex != 'firstName' && $fieldIndex != 'lastName') {
                    $import->fieldIndexes[$key] = $fieldIndexPrefix . $fieldIndex;
                }
            }
        }
 
        return $import;
    }
}
```

How to replicate it:
1. Go to contacts > Import
2. Choose the 'Import a user group and subscribe the users that belong to it to one or more mailing lists' way.
3. Click Next
4. Choose a mailing list and click 'Import'
5. The PHP warning appears

Thanks!

**Comment by bencroker** [comment]
Thanks for reporting this! Fixed in https://github.com/putyourlightson/craft-campaign/commit/34769ed771815fb8c7b2d2a80d260f2f7095143a for the next release.

You can test this by running `composer require "putyourlightson/craft-campaign:dev-develop as 2.9.4"`.

**Comment by bencroker** [comment]
Released in [2.9.4](https://github.com/putyourlightson/craft-campaign/releases/tag/2.9.4).
